sqlite/ext/comdb2/schistory.c

```diff
@@ -48,8 +48,9 @@ static int get_status(void **data, int *npoints)
     int rc, bdberr, nkeys;
     sc_hist_row *hist = NULL;
     struct sc_hist_ent *sc_hist_ents = NULL;
+    tran_type *trans = curtran_gettran();
```

Comment:
if (!trans) - error

Reply:
yes - the error check is also needed [here](https://github.com/bloomberg/comdb2/pull/2351/files#diff-1e5f67b586fef5a1387551dd66024cf5R197).